rules/REQUEST-920-PROTOCOL-ENFORCEMENT.conf

```diff
@@ -1150,6 +1150,31 @@ SecRule REQUEST_HEADERS_NAMES "@rx ^.*$" \
     chain"
     SecRule TX:/^header_name_/ "@within %{tx.restricted_headers}" \
         "setvar:'tx.anomaly_score_pl1=+%{tx.critical_anomaly_score}'"
+#
+# Rule against CVE-2022-21907
+# This rule blocks Accept-Encoding headers larger than 50 characters
+#
+# This rule has a stricter sibling: 920521
+#
+SecRule REQUEST_HEADERS:Accept-Encoding "@gt 50" \
+    "id:920520,\
+    phase:1,\
+    block,\
+    t:none,t:lowercase,t:length\
```

Comment:
@franbuehler Looks like a comma is missing here at the end :)

Reply:
Oh thank you for helping to find the lint error.
I'll fix that.